src/reviews/reviews.router.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const router = express.Router();
+const reviewsController = require('./reviews.controller')
+
+
+router.route('/reviews/product/:productid')
+    .get(reviewsController.getAllReviews)
+    .post(reviewsController.createReview)
+    .get(reviewsController.getReview)
+    .delete(reviewsController.removeReview)
+
+router.route('/reviews/user/:userid')
+    .getAll(reviewsController.getAllReviews)
+    .post(reviewsController.createReview)
+    .get(reviewsController.getReview)
+    .delete(reviewsController.removeReview)
+ 
+
```

Comment:
Falta otro router que reciba el id de review. Algo así como:


```suggestion
router.route('/reviews/:reviewid')
    .get(reviewsController.getReview)
    .delete(reviewsController.removeReview)
```